Tests/XCUITests/TabsPerformanceTests.swift

```diff
@@ -133,4 +139,65 @@ class TabsPerformanceTest: BaseTestCase {
             counter += 1
         }
     }*/
+
+    func testPerfHistory100startUp() {
+        waitForTabsButton()
+        app.terminate()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+            // activity measurement here
+            app.launch()
+            waitForTabsButton()
+        }
+    }
+
+    func testPerfHistory100openMenu() {
+        waitForTabsButton()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+                navigator.goto(LibraryPanel_History)
+                let loaded = NSPredicate(format: "count == 101")
+                expectation(for: loaded, evaluatedWith: app.tables[AccessibilityIdentifiers.LibraryPanels.HistoryPanel.tableView].cells, handler: nil)
+                waitForExpectations(timeout: TIMEOUT, handler: nil)
+        }
+    }
+
+    func testPerfBookmarks1000startUp() {
+        waitForTabsButton()
+        app.terminate()
+        measure(metrics: [
+            XCTMemoryMetric(),
+            XCTClockMetric(), // to measure timeClock Mon
+            XCTCPUMetric(), // to measure cpu cycles
+            XCTStorageMetric(), // to measure storage consuming
+            XCTMemoryMetric()]) {
+            // activity measurement here
+            app.launch()
+            waitForTabsButton()
+        }
+    }
+
+    func testPerfBookmarks1000openMenu() {
```

Comment:
This test works locally for me but fails on Bitrise:
```
testPerfBookmarks1000openMenu, Asynchronous wait failed: Exceeded timeout of 45 seconds, with unfulfilled expectations: "Expect predicate `count == 1001` for object <XCUIElementQuery: 0x600001124140>".
```
We may need to increase the timeout...

Reply:
looking at the screenshots, the bookmark list is populated, but for some reason the count of items is not available within that time